examples/user_interaction/client/src/App.js

```diff
@@ -23,47 +23,49 @@ class App extends React.Component {
     this.state = { pi: { time: 0, value: "unknown" }, prime_numbers: { time: 0, value: [] } };
     this.handleClick = this.handleClick.bind(this);
     this.runSecondTask = this.runSecondTask.bind(this);
-    this.http = require('http');
+    this.host = 'http://localhost:8088';
   }
 
   handleClick() {
-    const host = 'localhost';
-    const port = 8088;
-    this.http.get({
-      host,
-      port,
-      path: '/sleep'
-    }, (response) => {
-      let data = [];
-      response.on('data', chunk => data.push(chunk));
-      response.on('end', this.runSecondTask);
-    });
+    console.log("Entering handle click.");
+
+    this.runFirstTask();
+  }
+
+  runFirstTask() {
```

Comment:
What would you think about giving this a more desriptive name like `callSleepApi` or similar?